biztown/src/components/TableRow.js

```diff
@@ -1,96 +1,89 @@
-import React from "react"
-import {Label, Input} from 'reactstrap';
+import React from "react";
+import { Label, Input } from "reactstrap";
 
-const TableRow = (props) => {  
-  return(
+const TableRow = (props) => {
+  let questionId = "";
```

Comment:
take it or leave it pedantic nit: this seems like something that should be handled in the parent components. I like to think of each component as a modular  black box that has just enough information to perform it's function. In the code below, the child is assuming information about it's parent questionId which work functionally just fine, but in cases that aren't as particular as this, it would require more maintenance to keep the component functional.

Reply:
Definitely makes more sense for this to be passed from the parent component. Done